### DISCUSSION TITLE
delete the rust code

### DISCUSSION THREAD

**Discussion by RichardWright** [101arrowz/fflate#201] [Ideas]
The appearance of rust code in the codebase gives an impression that it's actually being used. Could it be removed?

**Comment by 101arrowz** [comment]
The Rust port was an experiment from a few years ago that never quite materialized. The README is quite explicit that the library is pure JavaScript. That being said, you are correct that it is not used, and it could potentially be misleading. I'll remove the Rust version with the next update.

**Reply by 101arrowz** [reply]
Deleted in v0.8.2.
